content/tutorials/_index.md

```diff
@@ -6,4 +6,4 @@ lastmod: 2021-03-19T17:03:04+01:00
 weight: 50
 cascade:
   type: tutorials
----
+---
```

Comment:
What was the change here? EOL that we cannot see?

Reply:
GitHub glitch, added and removed on same PR